rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accesor :date
+  attr_reader :book, :person
+
+  def initialize(date, _book, person)
+    @date = date
+    @book = bookbook.rentals << self
```

Comment:
- [ ] I believe you have 2 lines of code here but erroneously have it on one line:
```ruby
@book = book
book.rentals << self
```
Please implement this change